portafly/doc/ux_specs/audience/accounts/Listing.md

```diff
@@ -0,0 +1,21 @@
+# Audience > Accounts > Listing
+
+* [Mockups](https://marvelapp.com/55343de/screen/67190788)
```

Comment:
We also could consider to include them in our repo, so they don't depend on an external service. We could address this in a future PR

Reply:
I think it's better to keep the mockups in JIRA:
* they can be inspected and offer more relevant information than an image file
* they can be commented on the UI itself
* it's the accepted standard tool to deliver mockups across Red Hat